drivers/ieee802154/ieee802154_cc13xx_cc26xx.c

```diff
@@ -397,8 +397,16 @@ static void ieee802154_cc13xx_cc26xx_rx_done(
 			corr = drv_data->rx_data[i][len--] & 0x3F;
 			rssi = drv_data->rx_data[i][len--];
 
+			/* remove fcs */
+			if (IS_ENABLED(CONFIG_NET_L2_IEEE802154)) {
```

Comment:
Might be useful to use a similar structure that is used [here](https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/ieee802154/ieee802154_nrf5.c#L120). Both in terms of the comment and the conditions, as it might make later refactorizations slightly easier.